### PR TITLE
Apply pep8 rules in api and www/views [v2]

### PR DESCRIPTION
**Pull request by cacarrara**
Most of the changes are simple indentation and blank lines spaces. There was
also some import fixes that shuldn't change any behavior.

Probably the biggest change was in the method has_permission() of
PatchewPermission class (api/rest.py). Firstly the use of backslash inside
braces is redundant (to break lines). So in order to remove the backslashes
and trying to make the permission conditions more readable I've tried the
following approach.

It was also necessary to fix the `help_text` attributes of `Project` model fields.
The way the help text strings were being used in Project model
attributes was putting a lot of whitespaces and line breakes inside
them[1].

If these changes be accepted I'm willing to continue applying it on other files.

[1] - more details in commit message

---
Changes from v1[#95]:
- Squash commits related with PEP8 specifically;
- Fix minor bug added in `get_diff_stat` method from Message model;
- Fix help_text attribute from Project fields

### CODE REVIEW

**Comment by bonzini** [comment]
Looks good, I'll wait for Fam to give it another look and then push.

File: api/models.py [hunk 1]
```diff
@@ -501,7 +509,7 @@ def get_num(self):
                 try:
                     cur, total = int(n), int(m)
                     break
-                except:
+                except Exception:
                     pass
```
Comment (famz) [inline api/models.py]: Not related to this patch, but I think this should be return instead of pass.

File: www/views.py [hunk 2]
```diff
@@ -127,32 +138,33 @@ def gen_page_links(total, cur_page, pagesize, extra_params):
                 "title": str(i),
                 "url": "?page=" + str(i) + extra_params,
                 "class": "active",
-                "url": "#"
```
Comment (famz) [inline www/views.py]: I think the original idea and existing behavior is "#" but changing it to a link should be fine.

**Comment by famz** [comment]
@bonzini let's use the next branch from now on (which is synced to next.patchew.org).

@cacarrara Please rebase to next branch where there is a migration conflict.

File: www/views.py (hunk 2 above)
Comment (cacarrara) [reply www/views.py]: You can see the `url` key being defined twice here and we need only one. If you think it's better keep `#` I can change.

**Comment by cacarrara** [comment]
@famz rebased to next :+1:

**Pull request merged**